privacyidea/lib/policydecorators.py

```diff
@@ -628,39 +638,45 @@ def config_lost_token(wrapped_function, *args, **kwds):
             clientip = options.get("clientip")
             # get the policy
             policy_object = g.policy_object
-            contents_list = policy_object.get_action_values(
+            contents_dict = policy_object.get_action_values(
                 ACTION.LOSTTOKENPWCONTENTS,
                 scope=SCOPE.ENROLL,
                 realm=realm,
                 resolver=resolver,
                 user=username,
                 client=clientip,
-                unique=True)
-            validity_list = policy_object.get_action_values(
+                unique=True,
+                audit_data=g.audit_object.audit_data)
+            validity_dict = policy_object.get_action_values(
                 ACTION.LOSTTOKENVALID,
                 scope=SCOPE.ENROLL,
                 realm=realm,
                 resolver=resolver,
                 user=username,
                 client=clientip,
-                unique=True)
-            pw_len_list = policy_object.get_action_values(
+                unique=True,
+                audit_data=g.audit_object.audit_data)
+            pw_len_dict = policy_object.get_action_values(
                 ACTION.LOSTTOKENPWLEN,
                 scope=SCOPE.ENROLL,
                 realm=realm,
                 resolver=resolver,
                 user=username,
                 client=clientip,
-                unique=True)
+                unique=True,
+                audit_data=g.audit_object.audit_data)
 
-            if contents_list:
-                kwds["contents"] = contents_list[0]
+            if contents_dict:
+                g.audit_object.add_policy(contents_dict.values()[0])
+                kwds["contents"] = contents_dict.keys()[0]
 
-            if validity_list:
-                kwds["validity"] = int(validity_list[0])
+            if validity_dict:
+                g.audit_object.add_policy(validity_dict.values()[0])
```

Comment:
same as above

Reply:
Yes, you are right.
I will remove this.

Reply:
done